pm_symmetry/.shed.yml

```diff
@@ -7,5 +7,5 @@ long_description: |
   Given an input structure, generates a Wyckoff points symmetry report.
   Uses the pm-symmetry tool from pymuon-suite.
 name: pm_symmetry
-owner: muon-spectroscopy-computational-project
+owner: elichad
```

Comment:
is that on purpose?

Reply:
yes - we don't have a tool shed account for the group right now (no group email, etc). I initially thought whatever was here would show up as the 'owner' on the tool shed, but I realised while doing this that it needs to refer to a real account

Reply:
this owner will be part of every tool ID and you can not change it later, or its really hard.

A TS account can be any email address. It can also be changed later whenever you need to change it, but this owner-name will be persistent. For IUC its `iuc` for the Proteomics people its `galaxyp` and for the cheminformatics people its `chemteam`.

Reply:
Oh, I didn't realise it couldn't be changed. How does IUC adopt pre-existing tools then? Are they new repos with the same/another name, or do you do the 'really hard' bit?

I did think of that second part, but I'd already made a test repo manually that cemented my owner-name 😬 only on the test shed though

Reply:
> Oh, I didn't realise it couldn't be changed. How does IUC adopt pre-existing tools then? Are they new repos with the same/another name, or do you do the 'really hard' bit?

We ask the original author to provide IUC admin access to the repo under the old name. 

> I did think of that second part, but I'd already made a test repo manually that cemented my owner-name grimacing only on the test shed though

Mess around with the TTS as much as you like. The problem comes when you install tools, have them in workflows etc... then the tool ID is everywhere for reproducibility reasons and then you don't want to change it. So I recommend settling with a neutral name for your community and then start deploying to the main TS.

Reply:
Ok, I see. Thanks